desmume/src/frontend/posix/gtk/cheatsGTK.cpp

```diff
@@ -18,178 +18,237 @@
  * Boston, MA 02111-1307, USA.
  */
 
-
 #include <stdio.h>
 #include <stdlib.h>
 #include <gtk/gtk.h>
 #include "cheatsGTK.h"
 #include "cheatSystem.h"
+#include "utilsGTK.h"
 #include "main.h"
 #include "desmume.h"
 
 #undef GPOINTER_TO_INT
 #define GPOINTER_TO_INT(p) ((gint)  (glong) (p))
 
-enum {
+enum
+{
+    COLUMN_INDEX,
+    COLUMN_TYPE,
     COLUMN_ENABLED,
     COLUMN_SIZE,
     COLUMN_HI,
     COLUMN_LO,
+    COLUMN_AR,
     COLUMN_DESC,
     NUM_COL
 };
 
 enum
 {
-  COLUMN_SIZE_TEXT,
-  NUM_SIZE_COLUMNS
+    COLUMN_SIZE_TEXT,
```

Comment:
i'd really appreciate if whitespace changes could be kept to the bare minimum. if everyone reformats the code he's working on to his personal preferences, 90% of every commit is just visual noise when looking at the diffs.

Reply:
No problem, I'll do that first thing. I just try to leave code less of a mess than when I started.